docs/content/guide/getting-started.md

```diff
@@ -3,7 +3,7 @@ title: Getting started with Druxt.js
 weight: -9
 ---
 
-### Getting started with Druxt.js
```

Comment:
I specifically set this indentation to this level to make it more consistent with other parts of the site.

It should actually be remove entirely though, and dealt with on the Nuxt page component instead:
- https://github.com/druxt/druxt.js/blob/develop/docs/pages/guide/_.vue
- https://github.com/druxt/druxt.js/blob/develop/docs/pages/api/_.vue#L12